Ships as Raw.txt

```diff
@@ -177,7 +179,7 @@ RearArcConjunction|41
 51++
 Name|Victory I-class Star Destroyer

```

Comment:
It would ne *nice* if the ship entries were grouped by faction.  We start with Imperial, float through the prequal factions, and then came back to Imperial.